synapse/config/password.py

```diff
@@ -23,13 +23,17 @@ class PasswordConfig(Config):
     def read_config(self, config):
         password_config = config.get("password_config", {})
         self.password_enabled = password_config.get("enabled", True)
+        self.password_localdb = password_config.get("localdb", True)
```

Comment:
can we call this `password_localdb_enabled` or something slightly more intuitive?